Makefile

```
@@ -65,3 +65,13 @@ deb: build
 		--description "Boulder is an ACME-compatible X.509 Certificate Authority" \
 		--maintainer "$(MAINTAINER)" \
 		test/config/ sa/_db data/ $(OBJECTS) bin/ct-test-srv
+
+tar: build
+	fpm -f -s dir -t tar --name "boulder" \
+		--license "Mozilla Public License v2.0" --vendor "ISRG" \
+		--url "https://github.com/letsencrypt/boulder" --prefix=/opt/boulder \
+		--version "$(VERSION)" --iteration "$(COMMIT_ID)" --epoch "$(EPOCH)" \
+		--package "$(ARCHIVEDIR)/boulder-$(VERSION)-$(COMMIT_ID).amd64.tar.gz" \
+		--description "Boulder is an ACME-compatible X.509 Certificate Authority" \
+		--maintainer "$(MAINTAINER)" \
+		test/config/ sa/_db data/ $(OBJECTS)
```

Per the [conversation above](https://github.com/letsencrypt/boulder/pull/6302/files#r948456137) I believe you use the following block if you want a `.tar` file:

```suggestion
	fpm -f -s dir -t tar --name "boulder" --prefix=/opt/boulder \
		--package "$(ARCHIVEDIR)/boulder-$(VERSION)-$(COMMIT_ID).amd64.tar" \
		test/config/ sa/_db data/ $(OBJECTS)
		gzip -f "$(ARCHIVEDIR)/boulder-$(VERSION)-$(COMMIT_ID).amd64.tar"

```

or this block if you want `.tar.gz` file:

```suggestion
	fpm -f -s dir -t tar --name "boulder" --prefix=/opt/boulder \
		--package "$(ARCHIVEDIR)/boulder-$(VERSION)-$(COMMIT_ID).amd64.tar" \
		test/config/ sa/_db data/ $(OBJECTS)

```